repl/libjs.js

```diff
@@ -4272,8 +4272,8 @@ var dynCall_ji = Module["dynCall_ji"] = createExportWrapper("dynCall_ji");
 /** @type {function(...*):?} */
 var dynCall_jiji = Module["dynCall_jiji"] = createExportWrapper("dynCall_jiji");
 
-var ___start_em_js = Module['___start_em_js'] = 10075532;
-var ___stop_em_js = Module['___stop_em_js'] = 10075623;
+var ___start_em_js = Module['___start_em_js'] = 10076332;
+var ___stop_em_js = Module['___stop_em_js'] = 10076423;
```

Comment:
should this and the wasm regen be in a separate commit?

Reply:
ah, we don't actually need the libjs.* changes, I'll remove them from this PR.